public_html/wp-content/themes/pattern-directory/src/store/reducer.js

```diff
@@ -36,10 +36,16 @@ function queries( state = {}, action ) {
 	const patternIds = ( action.patterns || [] ).map( ( { id } ) => id );
 	const { page, total, totalPages } = action;
 	switch ( action.type ) {
-		case 'LOAD_BLOCK_PATTERNS':
+		case 'LOAD_BLOCK_PATTERNS': {
```

Comment:
It creates local scope so that I can use `const _queryState` in both case branches. I'm not suggesting it as practice going forward, just a workaround to keep the variables understandable.